game/scripts/vscripts/abilities/shielder/boss_shielder_shield.lua

```diff
@@ -7,7 +7,10 @@ function boss_shielder_shield:GetAbilityTextureName (brokenAPI)
 end
 
 function boss_shielder_shield:OnSpellStart()
-
+  local caster = self:GetCaster()
+  local particle = ParticleManager:CreateParticle("TheWarpiestOfShields.vpcf", PATTACH_ABSORIGIN_FOLLOW, caster)
+  ParticleManager:SetParticleControl(particle, 0, caster:GetAbsOrigin())
+  ParticleManager:SetParticleControl(particle, 1, caster:GetAbsOrigin())
```

Comment:
Probably leave the second control point for now since we don't know what we want it to control yet.